v4l2-request-test.c

```diff
@@ -306,9 +330,19 @@ int main(int argc, char *argv[])
 		case '?':
 			print_help();
 			goto error;
+		default:
+			printf("?? getopt returned character code 0%o ??\n", opt);
 		}
 	}
 
+	if (optind < argc) {
```

Comment:
We should probably remove this block too and just ignore non-option arguments.

Reply:
Yes. Got this from an example code and did experiment with it. But again, in our case we know all needed arguments (long and short) and can react as needed.